javatests/arcs/sdk/wasm/TestBase.kt

```diff
@@ -50,13 +50,18 @@ open class TestBase<T : WasmEntity>(
     }
 
     override fun fail(message: String?): Nothing {
+      fail(message, null)
+    }
+
+    @Suppress("VIRTUAL_MEMBER_HIDDEN") // Override for Kotlin/Native 1.4
```

Comment:
Nit: Can you put the comment under or over the suppress statement, not to the right? 
Otherwise, this change looks good.

Reply:
Moved above.